internal/module/kernelmapper.go

```diff
@@ -19,14 +19,14 @@ const (
 )
 
 type NodeOSConfig struct {
-	kernelFullVersion  string `subst:"KERNEL_FULL_VERSION"`
-	kernelVersionMMP   string `subst:"KERNEL_XYZ"`
-	kernelVersionMajor string `subst:"KERNEL_X"`
-	kernelVersionMinor string `subst:"KERNEL_Y"`
-	kernelVersionPatch string `subst:"KERNEL_Z"`
-	kernelRHELVersion  string `subst:"KERNEL_RHEL_VERSION"`
-	kernelRHELRelease  string `subst:"KERNEL_RHEL_RELEASE"`
-	kernelRHELArch     string `subst:"KERNEL_RHEL_ARCH"`
+	KernelFullVersion  string `subst:"KERNEL_FULL_VERSION"`
+	KernelVersionMMP   string `subst:"KERNEL_XYZ"`
+	KernelVersionMajor string `subst:"KERNEL_X"`
+	KernelVersionMinor string `subst:"KERNEL_Y"`
+	KernelVersionPatch string `subst:"KERNEL_Z"`
+	KernelRHELVersion  string `subst:"KERNEL_RHEL_VERSION"`
+	KernelRHELRelease  string `subst:"KERNEL_RHEL_RELEASE"`
+	KernelRHELArch     string `subst:"KERNEL_RHEL_ARCH"`
```

Comment:
How about we drop those fields and keep them for downstream only? We are about to push this code into Kubernetes.

Reply:
fixed